src/builder.rs

```diff
@@ -60,6 +60,7 @@ pub struct SessionBuilder {
     user: Option<String>,
     port: Option<String>,
     keyfile: Option<PathBuf>,
+    password: Option<String>,
```

Comment:
IMHO we can use [zeroize](https://docs.rs/zeroize/) here to avoid leaking the password.